### DISCUSSION TITLE
Atomstack A5 M50. Not the Pro version.

### DISCUSSION THREAD

**Discussion by RickR2023** [arkypita/LaserGRBL#2137] [Q&A]
I need the original firmware for the Atomstack A5 M50 to reset my machine. I can't find the firmware anywhere. Please help. Itried
X7V2022052301 firmware and it didn't work. Circles were distorted and writing too. Thank you, Rick

**Comment by Tribo-2dot0** [comment]
Pesquisando no google encontrei teu post no forum Lightburn e lá você diz que estava rodando bem... 
se usas uma versão crackeada de LB as formas e escritas vão sair distorcidas em determinado momento...
experimente usar com LaserGrbl.
